modules/runner/src/main/ts/core/Callbacks.ts

```diff
@@ -0,0 +1,72 @@
+import { noop } from './Utils';
```

Comment:
I think the separation of this file. Very clean.

Reply:
It'd be better if it didn't rely on a JQuery global (my one gripe) :stuck_out_tongue:

Reply:
Yeah, requiring jquery is unfortunate. Is it mainly just for the ajax function now?

Reply:
Are there any selectors beyond the capabilities of querySelector?

Reply:
Doubtful, it likely could be rewritten entirely in native JS now.